pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/model/config/ComputeAIEmbeddingsConfig.java

```diff
@@ -28,4 +35,10 @@ public class ComputeAIEmbeddingsConfig extends StepConfig {
 
   @JsonProperty(value = "embeddings-field", required = true)
   private String embeddingsFieldName;
+
+  @JsonProperty(value = "compute-service")
+  private String service;
+
+  @JsonProperty(value = "compute-config-json")
+  private String configJson;
```

Comment:
This is going to be really painful to configure from the CLI (JSON in JSON with all the escapings...)
Can we use the HuggingConfig directly instead ?

Reply:
most of the config (base URL for the API and the token) should be as a top level object, like "openai".

Here we keep only the other parameters specific to this step (like the "model" to use)